### PR TITLE
[zh] Update rbac.md

### PR DESCRIPTION
**Pull request by liutaot**

### CODE REVIEW

File: content/zh/docs/reference/access-authn-authz/rbac.md [hunk 1]
```diff
@@ -476,7 +476,7 @@ that change triggers adding the new rules into the aggregated ClusterRole.
 Here is an example that adds rules to the "monitoring" ClusterRole, by creating another
 ClusterRole labeled `rbac.example.com/aggregate-to-monitoring: true`.
 -->
-如果你创建一个与某现有聚合 ClusterRole 的标签选择算符匹配的 ClusterRole，
+如果你创建一个与某些有聚合 ClusterRole 的标签选择算符匹配的 ClusterRole，
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/rbac.md:479]:
```suggestion
如果你创建一个与某个已存在的聚合 ClusterRole 的标签选择算符匹配的 ClusterRole，
```

Comment (liutaot) [reply content/zh/docs/reference/access-authn-authz/rbac.md]: ok

**Comment by tengqm** [comment]
Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this **on the main branch**:

```shell
./scripts/lsync.sh content/zh/docs/reference/access-authn-authz/rbac.md
```

/hold

**Comment by howieyuen** [comment]
we do expect a fully synced pull request for doc repo, you can use command below to see details of latency:
```shell
./scripts/lsync.sh content/zh/docs/reference/access-authn-authz/rbac.md
```

**Pull request merged**